packages/react-app-rewire-flex-plugin/lib/index.js

```diff
@@ -47,6 +47,13 @@ function overrideWebpack(config, env) {
     '@twilio/flex-ui': 'flex-plugin/dev_assets/flex-shim.js',
   };
 
+  config.externals = {
```

Comment:
Let's describe what is happening in https://github.com/twilio/flex-plugin-builder/blob/master/packages/create-flex-plugin/templates/README.md

This file is added to the plugin project and would be useful for users to know that we are marking those libraries as externals and the actual versions in the package.json might not be used.

Reply:
Done!